hips/draw/simple.py

```diff
@@ -65,12 +29,52 @@ class SimpleTilePainter:
         HiPS survey properties
     tile : `HipsTile`
        An object of HipsTile
+    tile_format : `str`
+        Format of HiPS tile
     """
 
-    def __init__(self, geometry: WCSGeometry, hips_survey: HipsSurveyProperties, tile: HipsTile) -> None:
+    def __init__(self, geometry: WCSGeometry, hips_survey: HipsSurveyProperties, tile_format: str) -> None:
         self.geometry = geometry
         self.hips_survey = hips_survey
-        self.tile = tile
+        self.tile_format = tile_format
+
+    @property
+    def hips_order(self) -> int:
+        """Compute HiPS tile order matching a given image pixel size.
+
+        Examples
```

Comment:
This is a weird place to have the only example for this class. Move it to the class docstring (and maybe change it a bit, to show what properties are available after calling `run`.